main/tests.py

```diff
@@ -19,6 +21,11 @@ def create_dog_owner_user():
                            )
 
 
+@pytest.fixture()
+def create_request_factory():
+    return RequestFactory()
+
```

Comment:
can you rename the test methods? if someone does not know what is `RequestFactory` so this test is understandable